scripts/build-images

```diff
@@ -21,7 +21,7 @@ xargs -n1 -t docker image pull --quiet << EOF >> build/images-core.txt
     docker.io/rancher/klipper-helm:v0.4.3-build20210225
     docker.io/rancher/pause:3.2
     docker.io/rancher/nginx-ingress-controller-defaultbackend:1.5-rancher1
```

Comment:
Can drop this as it's not used with the newer chart.

Reply:
its just disabled in the new chart but if the user decided to enable it still needs the image

Reply:
We don't include it in 1.21 which is on the new-style chart, so either we need to re-add it there, or remove it from here.
https://github.com/rancher/rke2/blob/release-1.21/scripts/build-images#L24

Reply:
yeah I think I will readd it in 1.21 and master, and will add webhook image here